hikari/internal/routes.py

```diff
@@ -340,6 +340,10 @@ def compile_to_file(
 POST_CHANNEL_WEBHOOKS: typing.Final[Route] = Route(POST, "/channels/{channel}/webhooks")
 GET_CHANNEL_WEBHOOKS: typing.Final[Route] = Route(GET, "/channels/{channel}/webhooks")
 
+# Polls
+GET_ANSWER_VOTERS: typing.Final[Route] = Route(GET, "/channels/{channel}/polls/{message}/answer/{answer}")
+END_POLL: typing.Final[Route] = Route(POST, "/channels/{channel}/polls/{message}/expire")
```

Comment:
I think the name of `END_POLL` should be changed to `POST_END_POLL` to match the other methods.

Reply:
gotcha